cognite/seismic/protos/ingest_service_messages.proto

```diff
@@ -59,6 +59,10 @@ message RegisterFileRequest {
     **/
     google.protobuf.FloatValue source_group_scalar_override = 12;
     Dimensions dimensions = 13;                                     // [required] File data dimensionality, either 2D or 3D
+    google.protobuf.Int32Value cdp_offset = 14;                     // [optional] Ensemble (CDP) position in trace headers. Defaults to 21 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value energy_source_point_offset = 15;     // [optional] Energy source point position in trace headers. Defaults to 17 as per the SEG-Y rev1 specification
+    google.protobuf.Int32Value poststack_2d_shotpoint_offset = 16;  // [optional] Shotpoint for 2D poststack position in trace headers. Defaults to 197 as per the SEG-Y rev1 specification
```

Comment:
The `poststack_2d_` prefix seems a bit unneccesary to me, since all we support is poststack. Also it is possible these header field offsets will be used with 3d volume requests too since they share the same Trace proto

Reply:
The idea here is that it matches the name of the header in the SEGY spec. Since both offsets colloquially refer to shotpoints, it is good to use precise header names to distinguish them. Also, the slightly scary name might prompt users to read the documentation to see why it is like that!

Reply:
Although, looking at the segy spec again, it is really just called `shotpoint number`. :shrug: 

But using this offset for a 3D file should be an error, I think?

Reply:
Oh right. That is a conundrum. I'd vote for leaving out the irrelevant information, but it's a tradeoff.

Reply:
Right. Yeah..
Well, what are we landing on, then? :P This affects my other changes as well 😅

Reply:
I would say just `shotpoint_offset` fits in better with the other offset fields but I don't want to block this if I'm the only one disagreeing 😄

Reply:
I guess `shotpoint_offset` is fine. It's the name of the header.